### DISCUSSION TITLE
Reply all not available

### DISCUSSION THREAD

**Discussion by Vikash-P** [chatwoot/chatwoot#2941] [Q&A]
Team,
Do we have reply all feature

**Comment by Vikash-P** [comment]
Any luck please

**Reply by sojan-official** [reply]
could you provide more details? are you referring to the email channel?

**Comment by Vikash-P** [comment]
Hey Chatwoot/Chatwoot,

One of my team member has sent us mail with keeping some of my colleagues
over cc. However, when I replied it didn’t worked as reply all

**Reply by sojan-official** [reply]
we have this feature coming up in upcoming release : https://github.com/chatwoot/chatwoot/issues/2652

**Reply by Vikash-P** [reply]
Can i Please know release date of this please. Also, hope we are having feature to add some mail ids by default if we need to
